### ISSUE TITLE
Add support to prepend handler

### ISSUE THREAD

**Issue by filips123** [filp/whoops#587]
Add support to prepend handler. When using this, handler should be added before other handlers. Also, don't forget to update `RunInterface` with new function.

I found simple solution:
```php
public function prependHandler($handler)
{
    $existingHandlers = array_merge([$handler], $this->getHandlers());
    $this->clearHandlers();

    foreach ($existingHandlers as $existingHandler) {
        $this->pushHandler($existingHandler);
    }
}
```

**Comment by denis-sokolov** [comment]
The users of Whoops can perform this operation themselves if they need it. I believe most of the users will never encounter this situation, so I don’t think there’s a lot of valu in adding this feature. Change my mind if you have specific use cases!

**Comment by filips123** [comment]
I could be used if program first add some general handlers, but prepend additional handler under certain condition (maybe for additional debugging or other operations).

**Comment by denis-sokolov** [comment]
Thanks! I think for now we can afford to make the users perform the operation themselves. If we later see more use cases for this popup, we’ll revisit!

**Comment by filips123** [comment]
Ok. Do you have any solution how can users prepend handler themselves? My solution above could be inefficient because it needs to remove and then re-add all handlers.

**Comment by denis-sokolov** [comment]
Your solution above is absolutely adequate! I doubt you’ll measure a noticable impact on performance at all.

**Comment by denis-sokolov** [comment]
#630 has just added this, released in 2.4.0.
